sdk/openai/azure-ai-openai/TROUBLESHOOTING.md

```diff
@@ -123,6 +124,32 @@ for more information on why this happens and [ways to mitigate this issue][troub
 
 If you have network issues, please take a look at [troubleshooting network issues][troubleshooting_network_issues].
 
+### Configure RetryPolicy
+
+When receiving error code 423 `TooManyRequest`, the Azure OpenAI service will return a 24Hrs time window for the next request. 
+You can configure the `RetryPolicy` to handle this error. Here is an example of how to configure the `RetryPolicy`:
+
+```java 
+ExponentialBackoffOptions exponentialOptions = new ExponentialBackoffOptions()
+        .setMaxRetries(1) // The number of retries to attempt before giving up.
+        .setBaseDelay(Duration.ofSeconds(5)) // The base delay between retry attempts.
+        .setMaxDelay(Duration.ofSeconds(15)); // The maximum delay between retry attempts
+
+// Alternatively, FixedDelay retry strategy can be used to configure the RetryPolicy.
+// But here we are using ExponentialBackoff for demonstration.
+ExponentialBackoff backoffPolicy = new ExponentialBackoff(exponentialOptions);
+
+OpenAIClient client = new OpenAIClientBuilder()
+        .endpoint(endpoint)
+        .credential(new AzureKeyCredential(azureOpenaiKey))
+        .retryPolicy(new RetryPolicy(backoffPolicy, "{your-retry-header}", ChronoUnit.SECONDS))
```

Comment:
Use `retryOptions` builder method instead of `retryPolicy`

Reply:
Unfortunately, `retryOptions` doesn't have a way to set the [retryAfterHeader](https://github.com/Azure/azure-sdk-for-java/blob/10143a01f4c47ae351685c6c0b3d7b137e25aeb6/sdk/core/azure-core/src/main/java/com/azure/core/http/policy/RetryPolicy.java#L322-L331) value which will lead to call getWellKnownRetryDelay to favors the retry delay value from the response.